tests/extensions/config/resources/test_modifying_db_config.py

```diff
@@ -2,72 +2,245 @@
 # pylint: disable=missing-docstring
 import json
 
+from tests.extensions.config import utils
+
+
+def _patch_request(flask_app_client, data):
+    response = flask_app_client.patch(
+        '/api/v1/config/houston/',
+        content_type='application/json',
+        data=json.dumps(data),
+    )
+    return response
+
+
+def _test_op(value):
+    return {
+        'op': 'test',
+        'path': '/current_password',
+        'value': value,
+    }
+
+
+def _add_op(value, path='ENV'):
+    return {
+        'op': 'add',
+        'path': '/%s' % (path,),
+        'value': value,
+    }
+
+
+def _remove_op(path='ENV'):
+    return {
+        'op': 'remove',
+        'path': '/%s' % (path,),
+    }
+
+
+def test_modifying_db_config_by_regular_user(flask_app_client, regular_user, db):
+    try:
+        utils._get_and_check_houston_configs()  # Ensure an empty database of existing configs
+
+        with flask_app_client.login(regular_user, auth_scopes=('config.houston:write',)):
+            data = [
+                _test_op(regular_user.password_secret),
+                _add_op('testing-with-db'),
+            ]
+            response = _patch_request(flask_app_client, data)
+
+            assert response.status_code == 403
```

Comment:
This seems like it's something that should be very generic for many tests, maybe a function in test/utils.py where the status code is a parameter.

Reply:
Yes, I agree.  Let's add this to a subsequent PR to do some general clean-up.